dbt/include/trino/macros/materializations/seeds/helpers.sql

```diff
@@ -0,0 +1,85 @@
+{% macro trino__get_batch_size() %}
+  {{ return(1000) }}
+{% endmacro %}
+
+
+{% macro create_bindings(row, types) %}
+  {% set values = [] %}
+  {% set re = modules.re %}
+  
+  {%- for item in row -%}
+      {%- set type = types[loop.index0] -%}
+      {%- set match_type = re.match("(\w+)(\(.*\))?", type) -%}
+      {%- if item is not none and item is string and 'interval' in match_type.group(1) -%}
+        {%- do values.append((none, match_type.group(1).upper() ~ " " ~ item)) -%}
```

Comment:
I would argue that if we already defined the "interval" type for the column type,
the value specified should be

```
2
```

instead of 
```
'2' DAY
```

The logic here is then supposed to take the value and dress it into an `INTERVAL`.

This approach is less error prone for the user than the current approach. The user may attempt now to insert interval day into an interval month field.

Reply:
You can actually change the unit, for example '1' YEAR or '1' MONTH in the same type:

```
select TYPEOF(INTERVAL '1' YEAR);
select TYPEOF(INTERVAL '1' MONTH);
```

I think it's not complete up to par with for example snowflake, see https://docs.snowflake.com/en/sql-reference/data-types-datetime.html#interval-examples.

Reply:
I think it is much easier to avoid doing complex operations in interpreting values declared in a CSV file. Specifying the concrete value to be taken by the interval seems straightforward to me. We already know what kind of interval we are expecting (because we declared it in the column_types and therefore we can do easily the conversion.

Reply:
According to me it's less obvious: there are two types in trino for intervals, each of them supporting multiple unit types, by omitting the unit type, the user has to guess it as we didn't declare the unit type in the column_types. I think the unit types should be in the CSV eg. '1 day 2 minute 3 seconds', but for now this is also not possible because it requires setting up a combination of intervals eg INTERVAL '2' DAY + INTERVAL '3' SECOND. It' also not possible to combine YTM and DTS intervals. Actually doubting the usefulness of supporting it now.

- INTERVAL YEAR TO MONTH[#](https://trino.io/docs/current/language/types.html#interval-year-to-month)
Span of years and months.

Example: INTERVAL '3' MONTH

INTERVAL DAY TO SECOND[#](https://trino.io/docs/current/language/types.html#interval-day-to-second)
Span of days, hours, minutes, seconds and milliseconds.

Example: INTERVAL '2' DAY